src/Runner.Worker/JobExtension.cs

```diff
@@ -316,6 +316,29 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    if (message.Variables.TryGetValue("system.workflowFileFullPath", out VariableValue workflowFileFullPath))
+                    {
+                        context.Output($"uses: {workflowFileFullPath.Value}");
```

Comment:
```suggestion
                        context.Output($"Uses: {workflowFileFullPath.Value}");
```

Since all other lines are upper case first letters.